package.json

```
@@ -24,14 +24,14 @@
     "@types/mocha": "^8.2.3",
     "@types/node": "^16.11.13",
     "chai": "^4.3.4",
-    "circom": "^0.5.46",
-    "circomlib": "^0.5.5",
     "ethers": "^5.5.2",
     "hardhat": "^2.8.0",
     "mocha": "^9.1.3",
     "prettier": "2.3.2",
+    "circomlib": "^2.0.2",
+    "circomlibjs": "0.1.0",
```

I can reproduce the error if using 0.1.1
```suggestion
    "circomlibjs": "^0.1.1",
```